tests/phpt/handler_can_be_manually_called_multiple_times.phpt

```diff
@@ -17,7 +17,7 @@ $handler->exceptionHandler(new LogicException('terrible things'));
 --EXPECTF--
 array(1) {
   [0]=>
-  object(Exception)#15 (7) {
+  object(Exception)#%d (7) {
```

Comment:
These object IDs changed because we now instantiate the default callback classes — they should have always been "any number"